internal/artifacts/module_arch_test.go

```diff
@@ -784,11 +979,19 @@ func validateArchiveContents(expectedFilesInArchive []string, archiveLocation st
 	defer archiveReader.Close()
 	var filesInArchive []string
 	for _, file := range archiveReader.File {
-		filesInArchive = append(filesInArchive, file.Name)
+		if file.Mode().IsRegular() {
+			filesInArchive = append(filesInArchive, file.Name)
+		}
 	}
 	for _, expectedFile := range expectedFilesInArchive {
 		Ω(contains(expectedFile, filesInArchive)).Should(Equal(isExists), "Did not find "+expectedFile+" in archive")
 	}
+
+	if isExists {
```

Comment:
they were different before my change, this one had additional parameter.

Reply:
I know but they shouldn't be different, it's only duplicated because go doesn't allow to share test code in different packages. So we should keep them as similar as possible.
I think there is only 1 usage where `isExists` is false so maybe it's better to split it to 2 functions.

Reply:
done